migrations/2025-06-27-103657_post-or-comment-notification/up.sql

```diff
@@ -0,0 +1,107 @@
+-- create new data types
+CREATE TYPE notification_type_enum AS enum (
+    'Mention',
+    'Reply',
+    'Subscribed',
+    'PrivateMessage'
```

Comment:
Not exactly sure what subscribed is. You'd get a notif for subscribing to a community?

Reply:
Yes exactly, if you are subscribed to a post or community, notifications from there have the type `Subscribed`.